lutris/runners/wine.py

```diff
@@ -1178,6 +1236,11 @@ def get_env(self, os_env=False, disable_runtime=False):
         if not self.runner_config.get("dxvk") or not LINUX_SYSTEM.is_vulkan_supported():
             env["PROTON_USE_WINED3D"] = "1"
 
+        if self.runner_config.get("Graphics") == "wayland" \
```

Comment:
We're not setting ``PROTON_ENABLE_WAYLAND``- we should be. That's how to tell Proton to use the Wayland driver - not a registry key.

Reply:
Oops, yeah, totally forgot about that.

Reply:
Fixed, thanks